scripts/cybershake/install_cybershake_fault.py

```diff
@@ -229,31 +223,25 @@ def install_fault(
             ignore_vm_qpqs_files=ignore_vm_qpqs_files,
         )
 
+        if (
+            root_params_dict is None
+            or fault_params_dict is None
+            or sim_params_dict is None
+            or vm_params_dict is None
```

Comment:
Don't need to check vm_params_dict as load_yaml shouldn't be able to return None